frontend/src/schemas/party.schema.ts

```diff
@@ -0,0 +1,15 @@
+import { User } from '@/schemas/user.schema';
+
+export interface Party {
+	_id: string;
+	user: User;
+	category: 'Sinh nhật' | 'Đám cưới' | 'Khai trương' | 'Thôi nôi';
+	title: string;
+	description: string;
+	options: string[];
```

Comment:
type of option is false
follow:
options: Option[ ];

export interface Option {
	type: 'Basic' | 'Premium' | 'VIP';
	price: number;
}